src/components/ToDoList/ToDoList.test.tsx

```diff
@@ -3,33 +3,42 @@ import { ToDoList } from "./ToDoList";
 import * as toDoService from "../../services/toDoService";
 import { faker } from "@faker-js/faker";
 import userEvent from "@testing-library/user-event";
+import { ToDoItem } from "../../services/toDoItem";
 
 describe("ToDoList", () => {
     // true unit test
-    it("should load list items", () => {
+    it("should load list items", async () => {
         // Arrange
-        const toDoItems = [faker.lorem.word(), faker.lorem.word()];
-        jest.spyOn(toDoService, "getToDoList").mockReturnValue(toDoItems);
+        const toDoItems = createToDoItems(2);
+        jest.spyOn(toDoService, "getToDoList").mockImplementation(() => Promise.resolve(toDoItems));
 
         // Act
         render(<ToDoList />);
 
         // Assert
-        expect(screen.getByText(toDoItems[0])).toBeInTheDocument();
-        expect(screen.getByText(toDoItems[1])).toBeInTheDocument();
-    });
+        await(waitFor(() => {
+            expect(screen.getByText(toDoItems[0].description)).toBeInTheDocument();
+        }));
+
+        await (waitFor(() => {
+            expect(screen.getByText(toDoItems[1].description)).toBeInTheDocument();
+        }));
```

Comment:
Yes, I switched a lot of them over to findBy because the list component has to re-render after an add or delete, and the test was timing out. Guess I missed this one. Thank you!